src/mscorlib/src/System/Text/StringBuilder.cs

```diff
@@ -21,6 +21,7 @@ namespace System.Text {
     using System.Threading;
     using System.Globalization;
     using System.Diagnostics.Contracts;
+    using Collections.Generic;
```

Comment:
Why does this `using` not follow the same convention as the previous ones, i.e. to include `System` even when it's not necessary?